crypto/crypto-derive/src/lib.rs

```diff
@@ -368,6 +375,24 @@ pub fn hasher_dispatch(input: TokenStream) -> TokenStream {
         }
 
         impl libra_crypto::hash::CryptoHasher for #hasher_name {
+            fn seed() -> &'static [u8; 32] {
+                if let Some(seed_value) = #static_seed_name.get() {
```

Comment:
Why not `get_or_init`?
https://docs.rs/once_cell/1.4.0/once_cell/sync/struct.OnceCell.html#method.get_or_init

Reply:
:facepalm: ofc